src/widgets/WalletModal/ConnectModal.tsx

```diff
@@ -20,39 +20,34 @@ const HelpLink = styled(Link)`
   color: ${({ theme }) => theme.colors.sidebarColor};
 `;
 
-const ConnectModal: React.FC<Props> = ({ login, onDismiss = () => null }) => (
-  <Modal
-    title="Connect to a wallet"
-    bodyPadding="0 30px 30px 30px"
-    hideSeparator
-    showWalletBack
-    onDismiss={onDismiss}
-  >
-    {config.map((entry, index) => (
-      !window.ethereum && entry.redirectUrl ? (
-        <a key={entry.title} href={entry.redirectUrl} target="_blank" rel="noopener noreferrer">
-          <WalletCard
-            login={login}
-            walletConfig={entry}
-            onDismiss={onDismiss}
-            mb={index < config.length - 1 ? "8px" : "0"}
-          />
-        </a>
-      ) : (
+const ConnectModal: React.FC<Props> = ({ login, onDismiss = () => null }) => {
+  const walletsToShow = window.ethereum?.isTrust
+    ? config.filter((wallet) => wallet.title !== "WalletConnect")
+    : config;
```

Comment:
Why do we need to hide `WalletConnect` when it's on trust wallet browser?

Reply:
Bcs they can just click `Trust wallet` option adding Wallet connect as well is just confusing